integrationTesting/tests/organize/surveys/submitting-survey.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { expect } from '@playwright/test';
+
+import KPD from '../../../mockData/orgs/KPD';
+import KPDMembershipSurvey from '../../../mockData/orgs/KPD/surveys/MembershipSurvey';
+import RosaLuxemburg from '../../../mockData/orgs/KPD/people/RosaLuxemburg';
+import RosaLuxemburgUser from '../../../mockData/users/RosaLuxemburgUser';
+import test from '../../../fixtures/next';
+
+test.describe('User submitting a survey', () => {
+  test.afterEach(({ moxy }) => {
+    moxy.teardown();
+  });
+
+  test('submits data successfully', async ({ appUri, login, moxy, page }) => {
+    const apiPostPath = `/orgs/${KPDMembershipSurvey.organization.id}/surveys/${KPDMembershipSurvey.id}/submissions`;
+
+    moxy.setZetkinApiMock(
+      `/orgs/${KPDMembershipSurvey.organization.id}/surveys/${KPDMembershipSurvey.id}`,
+      'get',
+      KPDMembershipSurvey
+    );
+
+    moxy.setZetkinApiMock(
+      `/orgs/${KPDMembershipSurvey.organization.id}/surveys/${KPDMembershipSurvey.id}/submissions`,
+      'post',
+      {
+        timestamp: '1857-05-07T13:37:00.000Z',
+      }
+    );
+
+    const memberships = [
+      {
+        organization: KPD,
+        profile: {
+          id: RosaLuxemburg.id,
+          name: RosaLuxemburg.first_name + ' ' + RosaLuxemburg.last_name,
+        },
+        role: null,
+      },
+    ];
+
+    login(RosaLuxemburgUser, memberships);
+    await page.goto(
+      `${appUri}/o/${KPDMembershipSurvey.organization.id}/surveys/${KPDMembershipSurvey.id}`
+    );
+
+    await page.click('input[name="1.options"]');
+    await page.fill('input[name="2.text"]', 'Topple capitalism');
+    await page.click('input[name="sig"][value="authenticated"]');
+    await page.click('data-testid=Survey-acceptTerms');
+    await Promise.all([
+      page.waitForResponse((res) => res.request().method() == 'POST'),
+      await page.click('data-testid=Survey-submit'),
+    ]);
+
+    const reqLog = moxy.log(`/v1${apiPostPath}`);
+    expect(reqLog.length).toBe(1);
```

Comment:
I guess we should also verify here that the signature is being sent correctly. In this specific test, that would currently fail (I think) because only name/email signatures have been implemented so far and this test signs as the authenticated user.

Reply:
Fixed in 07d37bd7f32a113d8d53c7318e06e8d7bbc7e97b!